c++/test/tarray.cpp

```diff
@@ -459,7 +459,7 @@ test_array_info()
             for (ii = 0; ii < ndims; ii++)
                 if (rdims1[ii] != tdims1[ii]) {
                     TestErrPrintf(
-                        "Array dimension information doesn't match!, rdims1[%d]=%zd, tdims1[%d]=z%d\n", ii,
+                        "Array dimension information doesn't match!, rdims1[%d]=%llu, tdims1[%d]=z%llu\n", ii,
```

Comment:
Looks like that z is a typo?

Reply:
Good catch. I fixed one, but apparently there's this one too.